crates/xmtp-proto/src/gen/xmtp.keystore_api.v1.rs

```diff
@@ -0,0 +1,904 @@
+// @generated
```

Comment:
I could have sworn I `.gitignore`d these files. They don't really need to be checked in, given that they are recreated on every build.